amazon_eks/README.md

```diff
@@ -16,7 +18,20 @@ Monitoring EKS requires that you set up the Datadog integrations for:
 
 along with integrations for any other AWS services you're running with EKS (e.g., [ELB](https://docs.datadoghq.com/integrations/amazon_elb/))
 
-## Further Reading
+### Log Collection
+
+**Available for Agent >6.0**
+
+The setup is exactly the same as for Kubernetes. 
+To start collecting logs from all your containers, use your Datadog Agent [environment variables](https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/#log-collection-setup).
 
-Learn more about infrastructure monitoring and all our integrations on [our blog](https://www.datadoghq.com/blog/)
+Take also advantage of DaemonSets to [automatically deploy the Datadog Agent on all your nodes](https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/#log-collection-setup). 
```

Comment:
should the link be `https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/#container-installation`?

Reply:
@masci @l0k0ms Indeed that would seems more logical.

Reply:
updated :)